src/Symfony/Component/Console/Input/ArgvInput.php

```diff
@@ -44,8 +44,8 @@ class ArgvInput extends Input
     /**
      * Constructor.
      *
-     * @param array           $argv       An array of parameters from the CLI (in the argv format)
-     * @param InputDefinition $definition A InputDefinition instance
+     * @param array                $argv       An array of parameters from the CLI (in the argv format)
```

Comment:
@Ener-Getick I believe <code>array|null</code> is corrected here based on how the <code>@param</code> is written on line 48. Also, I think <code>array|null</code> matches the [Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html).